addons/survey/tests/test_survey_flow.py

```diff
@@ -1,29 +1,13 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-import re
-
 from odoo.addons.survey.tests import common
 from odoo.tests import tagged
 from odoo.tests.common import HttpCase
 
 
 @tagged('functional')
 class TestSurveyFlow(common.SurveyCase, HttpCase):
-
-    def _access_start(self, survey):
-        return self.url_open('/survey/start/%s' % survey.access_token)
-
-    def _access_page(self, survey, token):
-        return self.url_open('/survey/fill/%s/%s' % (survey.access_token, token))
-
-    def _access_submit(self, survey, token, post_data):
-        return self.url_open('/survey/submit/%s/%s' % (survey.access_token, token), data=post_data)
-
-    def _find_csrf_token(self, text):
-        csrf_token_re = re.compile("(input.+csrf_token.+value=\")([_a-zA-Z0-9]{51})", re.MULTILINE)
-        return csrf_token_re.search(text).groups()[1]
-
     def _format_submission_data(self, page, answer_data, additional_post_data):
```

Comment:
Couldn't we move this method to survey case as you use it in your certification tests ?

Reply:
I think I did it in a later commit.
Moved it to the right commit.